README.md

```diff
@@ -1,32 +1,186 @@
+    <a href="https://github.com/komodoplatform/atomicdex-api/issues?q=is%3Aissue+is%3Aclosed">
+        <img src="https://img.shields.io/github/issues-closed-raw/komodoplatform/atomicdex-api" alt="issues closed">
+    </a>
+    <a href="https://github.com/komodoplatform/atomicdex-api/pulls">
+        <img src="https://img.shields.io/github/issues-pr/komodoplatform/atomicdex-api" alt="pulls">
+    </a>
+    <a href="https://github.com/komodoplatform/atomicdex-api/pulls?q=is%3Apr+is%3Aclosed">
+        <img src="https://img.shields.io/github/issues-pr-closed/komodoplatform/atomicdex-api" alt="pulls closed">
+    </a>
+    <br/>
+    <a href="https://dev.azure.com/ortgma/Marketmaker/_build?definitionId=2">
+        <img src="https://img.shields.io/azure-devops/build/ortgma/marketmaker/2/mm2.1" alt="build status">
+    </a>
+    <a href="https://github.com/KomodoPlatform/atomicdex-api/releases">
+        <img src="https://img.shields.io/github/v/release/komodoplatform/atomicdex-api" alt="release version">
+    </a>
+    <a href="https://discord.gg/3rzDPAr">
+        <img src="https://img.shields.io/discord/412898016371015680?logo=discord" alt="chat on Discord">
+    </a>
+    <a href="https://twitter.com/intent/follow?screen_name=https://twitter.com/atomicdex">
+        <img src="https://img.shields.io/twitter/follow/atomicdex?style=social&logo=twitter" alt="follow on Twitter">
+    </a>
+</p>
 
-## Project structure
 
-[mm2src](mm2src) - Rust code, contains some parts ported from C `as is` (e.g. `lp_ordermatch`) to reach the most essential/error prone code. Some other modules/crates are reimplemented from scratch.
+## What is the AtomicDEX-API?
+
+The AtomicDEX API core is open-source [atomic-swap](https://komodoplatform.com/en/academy/atomic-swaps/) software for seamless, decentralised, peer to peer trading between almost every blockchain asset in existence. This software works with propagation of orderbooks and swap states through the [libp2p](https://libp2p.io/) protocol and uses [Hash Time Lock Contracts (HTLCs)](https://en.bitcoinwiki.org/wiki/Hashed_Timelock_Contracts) for ensuring that the two parties in a swap either mutually complete a trade, or funds return to thier original owner.
+
+There is no 3rd party intermediatary, no proxy tokens, and at all times users remain in sole possession of their private keys.
+
+A [well documented API](https://developers.komodoplatform.com/basic-docs/atomicdex/introduction-to-atomicdex.html) offers simple access to the underlying services using simple language agnostic JSON structured methods and parameters such that users can communicate with the core in a variety of methods such as [curl](https://developers.komodoplatform.com/basic-docs/atomicdex-api-legacy/buy.html) in CLI, [python scripts/TUIs](https://github.com/smk762/pytomicDEX_makerbot), or fully functioning [desktop and mobile applications](https://atomicdex.io/) such as [AtomicDEX Desktop](https://github.com/KomodoPlatform/atomicDEX-Desktop).
```

Comment:
I think maybe it is better to create a shortlist with interfaces examples powered by KomodoPlatform and community-powered ones? (TUIs, scripts, ...)
This way anyone will be able to add his interface, e.g. Roman also developed TUI for mm2 but it not mentioned here: https://github.com/Milerius/mm2-client

Reply:
> I think maybe it is better to create a shortlist with interfaces examples powered by KomodoPlatform and community-powered ones? (TUIs, scripts, ...) This way anyone will be able to add his interface, e.g. Roman also developed TUI for mm2 but it not mentioned here: https://github.com/Milerius/mm2-client

Would be nice to mention https://github.com/Milerius/awesome-atomicdex then

Reply:
> Would be nice to mention https://github.com/Milerius/awesome-atomicdex then

It's worth transferring it to the Komodo Platform Github and pinning.

Reply:
> > Would be nice to mention https://github.com/Milerius/awesome-atomicdex then
> 
> It's worth transferring it to the Komodo Platform Github and pinning.

done

Reply:
link to Awesome AtomicDEX added